sdk/keyvault/azure-security-keyvault-jca/src/test/java/com/azure/security/keyvault/jca/PropertyConvertorUtils.java

```diff
@@ -17,7 +18,8 @@
     public static void putEnvironmentPropertyToSystemPropertyForKeyVaultJca() {
         KEYVAULT_JCA_SYSTEM_PROPERTIES.forEach(
             environmentPropertyKey -> {
-                String value = System.getenv(environmentPropertyKey);
+                String value = Configuration.getGlobalConfiguration()
+                    .get(environmentPropertyKey, System.getenv(environmentPropertyKey));
```

Comment:
How about adding a new method in `PropertyConvertorUtils`:
```
    public static final String getPropertyValue(String property) {
        return Configuration.getGlobalConfiguration().get(property, System.getenv(property));
    }
```

And use it in all places?

Reply:
Fixed in new version.